pkg/cmd/workspace/delete.go

```diff
@@ -113,8 +114,24 @@ var DeleteCmd = &cobra.Command{
 		if !yesFlag {
 			fmt.Println("Operation canceled.")
 		} else {
+			if !forceFlag {
```

Comment:
There shouldn't be a prompt to force remove the workspace. This is something that should exclusively be controlled by the `-f` flag.

On a side note: `Provider resources might not be removed if you don't use this option` => this is not correct. Provider resources might not be removed if you **do** use this option.

I suggest removing this prompt entirely.
@idagelic do you agree?